scripts/uitest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+# Define paths to uitest directories and command.
+script_dir=`dirname $0`
+root_dir=`realpath $script_dir/../`
+uitest_dir=$root_dir/uitest
+uitest=$uitest_dir/devenv/bin/uitest
+
+# Define default parameters.
+cloud="lxd"
+archive=`ls $root_dir/dist/jujugui-*.tar.bz2`
```

Comment:
No because 'dist' does a 'clean-dist'.  (I just checked because I had the same worry.)